### PR TITLE
Updated build to automatically switch to JDK 8 if env is available

### PR DESCRIPTION
**Pull request by collado-mike**
This enables building the entire project with a single gradle command, making it easier to run all tests before committing

### CODE REVIEW

File: integrations/spark/src/test/java/marquez/spark/agent/lifecycle/LibraryTest.java [hunk 1]
```diff
@@ -1,11 +1,7 @@
 package marquez.spark.agent.lifecycle;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.fail;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.reset;
-import static org.mockito.Mockito.when;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```
Comment (wslulciuc) [inline integrations/spark/src/test/java/marquez/spark/agent/lifecycle/LibraryTest.java:4]: _Minor:_ As part of our style guide, we avoid wildcards `*`. See, https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

**Review by wslulciuc — APPROVED** [review]
Nice work, @collado-mike! Left a minor comment, otherwise 👍

**Comment by wslulciuc** [comment]
@collado-mike: Looks like you'll also need to sign your commits :)

File: integrations/spark/src/test/java/marquez/spark/agent/lifecycle/LibraryTest.java (hunk 1 above)
Comment (collado-mike) [reply integrations/spark/src/test/java/marquez/spark/agent/lifecycle/LibraryTest.java:4]: Hmm, thanks. Intellij must have done that automatically. 

If you don't mind, can we just codify these kinds of style preferences into the style checker? I don't like wasting PR comments on things that can/should be caught by a linter. It's also a waste of build cycles, as I'll now end up modifying the commit, pushing to origin, waiting for circle,... I think spotless probably has a rule to disallow * imports, so when I run `gradle build` (for the whole project 😁), it'll catch these issues before I push. Sound ok? I can just update the style checker as part of this PR (unless you prefer a separate one).

Comment (wslulciuc) [reply integrations/spark/src/test/java/marquez/spark/agent/lifecycle/LibraryTest.java:4]: Make sense. It's cool to merge without the style change. I've been meaning to add a pre-commit hook that checks the code for styling (see https://github.com/MarquezProject/marquez/issues/906) and will get to it this some time this week.

**Pull request merged**